src/Keboola/DbWriter/Writer/Pgsql.php

```diff
@@ -412,6 +413,9 @@ private function getStageColumnDataTypeSql(array $columnDefinition)
         if (strtolower($columnDefinition['type']) === 'text') {
             return 'TEXT';
         }
+        else if (strpos($columnDefinition['type'], '[]') !== false) {
```

Comment:
Kdyz bude mist ve zdrojovych datech prazdny string `""` tak tohle nebude fungovat.

Reply:
ale to je asi na nich prehnat si to niecim (transformaciou), co tam da `{}`, nie?

Reply:
Jasne, ale ten `INTEGER[]` muze byt i NULLABLE a pak chces nase `""` dostat do DB jako `NU""`

Reply:
Ty array types maji "unlimited" delku, ale to ma i text. Takze kdyz tenhle radek se vyhodi, tak to imho bude fungovat. Dobry ale to doplnit do testu.

Reply:
pockat, to potom ale staci doplnit test, kedy bude ten stlpec nullable, nie?

btw, neches to v ramci review rovno dotiahnut? :)